app/partials/analysis/charts/sidebar.jade

```diff
@@ -1,17 +1,16 @@
-div(ng-controller="chartsSidebarCtrl")
-    form
-        fieldset
-            legend Project
-            label fork&nbsp;
-                input(ng-model="currentFork",disabled = true).input-small
-            legend Plots
-            label Choose one&nbsp;
-                select(ng-model="currentPlot")
-                    option(ng-repeat='p in plots') {{p.label}}
-            legend Variables
-            label X&nbsp;
-            select(ng-model="xVariable")
-                option(ng-repeat="v in variables") {{v}}
-            label Y&nbsp;
-            select(ng-model="yVariable")
-                option(ng-repeat="v in variables") {{v}}
+div(ng-controller="sidechartsCtrl")
+  h4 Graphs
+  select.form-control#chooseGraph(value="Choose a graph" ng-change="changeName()" ng-model = "graphSelect" ng-options = "g as g.name for g in graphs")
```

Comment:
(style): unify usage of spaces: they should always be present after comma, but not around equality sign when you pass parameter/attribute values: i.e. `<ng-model="graphSelect">`, etc.